Utils/utils.py

```
@@ -75,36 +118,129 @@ def send_chunks(send_func, channel, text: str, chunk_size: int = CHUNK_SIZE) ->
-
-    if config.PROXY:
+    if config.PROXY and not bypass_proxy:
         target_url = config.PROXY
         headers["url"] = url
+    res = requests.request(method, target_url, headers=headers, timeout=(timeout, timeout), **kwargs)
 
-    res = requests.get(target_url, headers=headers, timeout=(timeout, timeout))
     return res
 
-def proxy_post_request(url: str, data=None, json=None, headers=HEADERS, timeout=TIMEOUT) -> requests.Response:
+def fetch_firefox_cookies(domains: List[str] = [], as_netscape: bool = False) -> Union[str, Dict[str, str]]:
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)